lib/improver/tests/utilities/cube_manipulation/test__equalise_cube_attributes.py

```diff
@@ -149,7 +150,8 @@ def test_unknown_attribute(self, warning_list=None):
 
         cubelist = iris.cube.CubeList([cube1, cube2])
 
-        result = _equalise_cube_attributes(cubelist)
+        result = _equalise_cube_attributes(
+            cubelist)
```

Comment:
Why has this found itself on a new line? Also needs the keyword.

Reply:
Fixed the new line. If the keyword is specified here the test fails because it initially checks that a `UserWarning` is generated for a mismatching attribute `'Do not know what to do with mosg__model_configuration etc...`' found on the cubes whereas if I provide it here it matches this and doesn't raise the UserWarning.

Reply:
I think you should have ignored me 😢 
I don't think this is needed here, indeed we want to test that the keyword argument it not mandatory, so it would probably be best to put it back the way it was. Apologies.

Reply:
I've undone this change. Sorry again Mark.